critiquebrainz/frontend/templates/artist/entity.html

```diff
@@ -181,3 +194,25 @@
   </div>
 </div>
 {% endblock %}
+
+{% block scripts %}
+  {{ super() }}
+  <script>
+    $(document).ready(function() {
+      $("#only-reviewed").change(function() {
+        if($('#only-reviewed').is(":checked")) {
+          $(".not_reviewed").hide();
+          var reviewed = $('.reviewed').length;
+          var release_group_count = $('.release-group').length;
+          if(reviewed == 0 && release_group_count != 0) {
+              $("#no-reviewed-entities").removeClass('hidden');
+          }
+        }
+        else {
+          $(".not_reviewed").show();
+          $("#no-reviewed-entities").addClass('hidden');
+        }
+      })
+    })
+  </script>
```

Comment:
It's better to put this JS code into a separate script in *static/scripts*, build and import it.